Source/UIView+draggable.h

```diff
@@ -77,4 +77,11 @@
  */
 - (void)setDraggable:(BOOL)draggable;
 
+/** Enables the dragging
+ *
+ * Enables move view to the front
+ */
+
+-(void)setMoveable:(BOOL)moveable;
```

Comment:
I think we should use a more clear term. "Move to front" or something like that.